chef/data_bags/crowbar/bc-template-cinder.json

```diff
@@ -19,6 +19,9 @@
       "use_gitbarclamp": true,
       "use_virtualenv": true,
       "use_pip_cache": true,
+      "min_pool_size": 10,
+      "max_pool_size": 100,
+      "pool_timeout": 2,
```

Comment:
* Hard to say what's a good default, it depends on the usage of the DB that depends on how you are using the cloud. If you are running stress tests of course these values need to be high. The values that are in the upstream config files are pretty low and usually not usable in a real production environment. If i good remember I took the values here from an upstream bug report and they solved the issue the customer was seeing. Maybe they are a bit an overkill in a normal setup...
* In Nova would make sense...I think that's the only other project that is pretty chatty with the DB 
* I don't think it's wrong to have different default values for different project since some project stress the DB more than others. In this case anyay I think I can use the same values we set for Neutron, they seem less extreme. Agreed?